src/remote.go

```diff
@@ -142,11 +142,7 @@ func (r *Remote) changes(startChangeId int64) (chan *drive.Change, error) {
 func buildExpression(parentId string, typeMask int, inTrash bool) string {
 	var exprBuilder []string
 
-	if inTrash || (typeMask&InTrash) != 0 {
```

Comment:
Aha I see, so this simplifies the trashed condition checks. Does it work alright with `list` as well as `--trashed` in nested directories, not just root?